maventa/config.ts

```diff
@@ -0,0 +1,16 @@
+import dotenv from 'dotenv';
```

Comment:
Do not introduce a dependency for dotenv. 

Our Process library has this .env support already.

Reply:
See https://github.com/hyperifyio/io.hyperify.core/?tab=readme-ov-file#processutilsinitenvfromdefaultfiles